### PR TITLE
Section C: Encounter Data Report

### PR DESCRIPTION
**Pull request by karla-vm**
## Description

Closes [MDCT-425](https://qmacbis.atlassian.net/browse/MDCT-425).

Adding Section C: Encounter Data Report to MCPAR report.

### How to test

- `./dev local`
- Navigate to /mcpar/program-level-indicators/encounter-data-report

### Changed Dependencies

N/A

## Code author checklist
- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [ ] I have added analytics, if necessary
- [ ] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Review by juliuscaesar — APPROVED** [review]
Looks good to me!

**Review by braxex — APPROVED** [review]
Looks great. Just needs the changes from https://github.com/CMSgov/mdct-mcr/pull/3460 to work. Wait until that's merged, then pull, then you're good to merge. 👍

**Pull request merged**
